coral-component-steplist/src/scripts/Step.js

```diff
@@ -140,26 +140,66 @@ class Step extends BaseComponent(HTMLElement) {
   /**
     Reflects the <code>aria-label</code> attribute to the marker dot for cases where no visible label is provided for the Step.
     @type {?String}
-    @default ''
+    @default null
     @htmlattribute labelled
-    @htmlattributereflected
     @memberof Coral.Step#
   */
   get labelled() {
-    return this._labelled || this.getAttribute('labelled') || this._elements.stepMarkerContainer.getAttribute('aria-label') || '';
+    return this._elements.stepMarkerContainer.getAttribute('aria-label');
   }
   set labelled(value) {
-    this._labelled = transform.string(value);
```

Comment:
With these changes, it seems the 'labelled' attribute won't be in sync with 'labelled' property. 
Is it true?

Reply:
I'm following the example set by `BaseFormField`: https://github.com/adobe/coral-spectrum/pull/117/files/6b1f018d581da504416f37c833f8d105aeecaf25#diff-0fb7dd9e799b24eb8adff0927156792fR170-R184, because I don't want an empty `labelled` or `labelledby` attributes on Step when the value is `null`.